pkg/components/asap.go

```diff
@@ -115,9 +115,16 @@ func ASAPToken(_ context.Context, _ string, _ string, _ string) (interface{}, er
 	return &ASAPTokenComponent{}, nil
 }
 
+// NewComponent populates an ASAPToken with defaults.
+func NewComponent() *ASAPTokenComponent {
+	return &ASAPTokenComponent{}
+}
+
 // Settings generates a config populated with defaults.
 func (m *ASAPTokenComponent) Settings() *ASAPTokenConfig {
-	return &ASAPTokenConfig{}
+	return &ASAPTokenConfig{
+		TTL: time.Minute * -60,
```

Comment:
Is the negative here intentional?

Reply:
LOL.  That was me doing a manual test and being too fast.  Good eyes.  I'll fix.  :shame:

Reply:
Agreed with @akhalina with the negative hahaha